### PR TITLE
Accept watchman's full match expression

### PR DESCRIPTION
**Pull request by Liooo**
Accepts [watchman's match expression](https://facebook.github.io/watchman/docs/expr/match.html) when array is given in `watch` options items.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -120,7 +120,12 @@ class ServerlessAppSyncSimulator {
       const sub = {
         expression: [
           'anyof',
-          ...this.options.watch.map((glob) => ['match', glob]),
+          ...this.options.watch.map((glob) => {
+            if (Array.isArray(glob)) {
+              return glob;
+            }
+            return ['match', glob];
```
Comment (Liooo) [inline src/index.js:127]: `glob` is expected to be string if not array, and this line is added for backward compatibility.  Not validating types other than above in this PR.

**Review by bboure — APPROVED** [review]
Thank you @Liooo  awesome work

**Comment by bboure** [comment]
@semantic-release-bot Add @Liooo for code and doc

**Comment by bboure** [comment]
@all-contributors Add @Liooo for code and doc

**Pull request merged**